conf/broker.conf

```diff
@@ -148,6 +148,15 @@ maxUnackedMessagesPerBroker=0
 # limit/2 messages
 maxUnackedMessagesPerSubscriptionOnBrokerBlocked=0.16
 
+# Subscribe too much for a consumer and load data from bookie to cause the high network bandwidth usage.
```

Comment:
`too many subscribe requests from a consumer can cause broker rewinding consumer cursors and loading data from bookies, hence causing high network bandwidth usage`